Test.AspNetCore.ServerSentEvents/Middleware/ValidationTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Authorization.Policy;
+using Moq;
+using Xunit;
+using Lib.AspNetCore.ServerSentEvents;
+using Test.AspNetCore.ServerSentEvents.Middleware.Infrastructure;
+
+namespace Test.AspNetCore.ServerSentEvents.Middleware
+{
+    public class ValidationTests
```

Comment:
I'm wondering (but not sure) if those are in fact additional HandshakeTests

Reply:
My reasoning for this was that Handshake handles the SSE bit, and this occurs just before something is decided to be SSE in the first place.